### ISSUE TITLE
Conda update fails

### ISSUE THREAD

**Issue by tb7554** [conda/conda#4053]
Getting the same error message for any update command:

conda update conda
Fetching package metadata ...An unexpected error has occurred.
Please consider posting the following information to the
conda GitHub issue tracker at:

    https://github.com/conda/conda/issues

Current conda install:

               platform : osx-64
          conda version : 4.2.13
       conda is private : False
      conda-env version : 4.2.13
    conda-build version : 1.19.0
         python version : 2.7.12.final.0
       requests version : 2.12.3
       root environment : /Users/me/anaconda  (writable)
    default environment : /Users/me/anaconda
       envs directories : /Users/me/anaconda/envs
          package cache : /Users/me/anaconda/pkgs
           channel URLs : https://conda.anaconda.org/rwest/osx-64
                          https://conda.anaconda.org/rwest/noarch
                          https://conda.anaconda.org/vgauthier/osx-64
                          https://conda.anaconda.org/vgauthier/noarch
                          https://repo.continuum.io/pkgs/free/osx-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/pro/osx-64
                          https://repo.continuum.io/pkgs/pro/noarch
            config file : /Users/me/.condarc
           offline mode : False

`$ /Users/me/anaconda/bin/conda update conda`

    Traceback (most recent call last):
      File "/Users/me/anaconda/lib/python2.7/site-packages/conda/exceptions.py", line 479, in conda_exception_handler
        return_value = func(*args, **kwargs)
      File "/Users/me/anaconda/lib/python2.7/site-packages/conda/cli/main.py", line 145, in _main
        exit_code = args.func(args, p)
      File "/Users/me/anaconda/lib/python2.7/site-packages/conda/cli/main_update.py", line 65, in execute
        install(args, parser, 'update')
      File "/Users/me/anaconda/lib/python2.7/site-packages/conda/cli/install.py", line 238, in install
        prefix=prefix)
      File "/Users/me/anaconda/lib/python2.7/site-packages/conda/api.py", line 24, in get_index
        index = fetch_index(channel_urls, use_cache=use_cache, unknown=unknown)
      File "/Users/me/anaconda/lib/python2.7/site-packages/conda/fetch.py", line 300, in fetch_index
        repodatas = [(u, f.result()) for u, f in zip(urls, futures)]
      File "/Users/me/anaconda/lib/python2.7/site-packages/concurrent/futures/_base.py", line 403, in result
        return self.__get_result()
      File "/Users/me/anaconda/lib/python2.7/site-packages/concurrent/futures/thread.py", line 55, in run
        result = self.fn(*self.args, **self.kwargs)
      File "/Users/me/anaconda/lib/python2.7/site-packages/conda/fetch.py", line 75, in func
        res = f(*args, **kwargs)
      File "/Users/me/anaconda/lib/python2.7/site-packages/conda/fetch.py", line 117, in fetch_repodata
        timeout=(6.1, 60))
      File "/Users/me/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 501, in get
        return self.request('GET', url, **kwargs)
      File "/Users/me/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 488, in request
        resp = self.send(prep, **send_kwargs)
      File "/Users/me/anaconda/lib/python2.7/site-packages/requests/sessions.py", line 609, in send
        r = adapter.send(request, **kwargs)
      File "/Users/me/anaconda/lib/python2.7/site-packages/requests/adapters.py", line 423, in send
        timeout=timeout
      File "/Users/me/anaconda/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py", line 594, in urlopen
        chunked=chunked)
      File "/Users/me/anaconda/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py", line 350, in _make_request
        self._validate_conn(conn)
      File "/Users/me/anaconda/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py", line 835, in _validate_conn
        conn.connect()
      File "/Users/me/anaconda/lib/python2.7/site-packages/requests/packages/urllib3/connection.py", line 330, in connect
        cert = self.sock.getpeercert()
      File "/Users/me/anaconda/lib/python2.7/site-packages/requests/packages/urllib3/contrib/pyopenssl.py", line 324, in getpeercert
        'subjectAltName': get_subj_alt_name(x509)
      File "/Users/me/anaconda/lib/python2.7/site-packages/requests/packages/urllib3/contrib/pyopenssl.py", line 171, in get_subj_alt_name
        ext = cert.extensions.get_extension_for_class(
    AttributeError: 'Extensions' object has no attribute 'get_extension_for_class'

**Comment by mbaum1122** [comment]
I just experienced this same error

**Comment by aliceell** [comment]
I have the same problem.

**Comment by aliceell** [comment]
I found a solution here: #3938

It looks like this is a conda issue (as I suspected), not a Spyder issue. Something to do with multiple dependencies.

The CONDA_SSL_VERIFY command did not work for me. I had to use the following commands as suggested by [this comment](https://github.com/conda/conda/issues/3938#issuecomment-263755414):

```
conda config --set ssl_verify false
conda update pyopenssl
conda config --remove ssl_verify
```

Hope this helps someone else.
